shared/app.js

```diff
@@ -73,7 +73,7 @@ module.exports = Backbone.Model.extend({
      * Initialize the `ClientRouter` on the client-side.
      */
     if (!isServer) {
-      var ClientRouter = this.options.ClientRouter || require(defaultRouterModule);
+      var ClientRouter = this.options.ClientRouter || this.modelUtils._requireAMD(defaultRouterModule);
```

Comment:
Yes, it was working on `1.0.1`  with `ClientRouter = require('app/router');`.

I don't think PR is going to fix the issue